### DISCUSSION TITLE
Basics // Table is influencing print output height

### DISCUSSION THREAD

**Discussion by LeHarry** [frappe/print_designer#174] [Q&A] [answered]
If number of items < tablesize:

![grafik](https://github.com/frappe/print_designer/assets/160015230/1a991be1-0ea7-41ba-a5fd-50b561d8e656)
![grafik](https://github.com/frappe/print_designer/assets/160015230/8604deee-734c-4f1d-a5db-93df10b274f3)

If number of items > tablesize:

![grafik](https://github.com/frappe/print_designer/assets/160015230/9d652b9f-d6cc-454f-ab5d-aef1703e0bbb)
![grafik](https://github.com/frappe/print_designer/assets/160015230/4b896347-a28f-4806-8964-e465802eae2e)

**Comment by maharshivpatel** [comment, marked as answer]
@LeHarry  If you want fixed location you have to make it part of footer. it also means they will be repeated on all pages. https://youtu.be/XpmrLzbianE?si=eetZkK0KqdrmGLCd&t=749
